docker/bookkeeper/entrypoint.sh

```diff
@@ -32,7 +20,11 @@ BK_CLUSTER_NAME=${BK_CLUSTER_NAME:-"bookkeeper"}
 BK_LEDGERS_PATH="/${PRAVEGA_PATH}/${PRAVEGA_CLUSTER_NAME}/${BK_CLUSTER_NAME}/ledgers"
```

Comment:
Do you know where PRAVEGA_CLUSTER_NAME set ?

Reply:
It is here: 

https://github.com/pravega/pravega/blob/e2dfc4681721b4576a090e33e09f79d24040ddc2/docker/bookkeeper/entrypoint.sh#L18. The default value is `pravega-cluster`.